crates/conf/src/conf.rs

```diff
@@ -220,9 +233,17 @@ impl AudioConf {
     }
 }
 
+impl Window {
+    /// Window mode.
+    pub fn mode(&self) -> WindowMode {
+        self.mode
+    }
+}
+
 // Bundle configuration neatly into a single struct
 bundle_config!(
     camera: CameraConf: Camera, // Conf file -> Camera -> CameraConf
     multiplayer: MultiplayerConf: MultiplayerConf,  // Conf file -> MultiplayerConf
-    audio: AudioConf: AudioConf
+    audio: AudioConf: AudioConf,
+    window: Window: Window
```

Comment:
I would place a comma at the end of line (if the macro allows) so that in the future, the diff has only one line (unlike now).